test/test_ui_commands.py

```diff
@@ -96,10 +96,10 @@ def setUp(self):
     def tearDown(self):
         self.io.restore()
 
-    def remove_keys(self, l, text):
+    def remove_keys(self, lrm, text):
```

Comment:
Perhaps a better name for this argument would be `lst` (the `l` is short for "list").